localip_test.go

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2015 Uber Technologies, Inc.
```

Comment:
Keeping it 2015 to keep it consistent with the rest of the code, apparently the "year of first publication" is what we want:
http://www.contentious.com/2007/01/07/copyright-notice-is-the-year-really-necessary/